### ISSUE TITLE
GeoJSON files reading

### ISSUE THREAD

**Issue by drish** [k255/drill-gis#4]
hey @k255 ,

first of all, thanks for this great work on this plugin.

It is probably not part of the scope of this project but, I noticed that all samples and examples you're using are querying from CSV files.

I wonder if this plugin has any type of helper on reading/parsing/transforming geo.json files on drill, since the native drill json reader doesn't support lists(json arrays) of multiple types, thus failing on reading complex geojson files.

I found on this thread http://drill-user.incubator.apache.narkive.com/d45QmsiL/passing-json-map-as-string-to-udf that you had the same problem initially trying to read geo.json files, but I couldn't find any further information on how you dealt with this beyond that thread.

I've also tried setting drill.exec.union_types as true , but basically it returns the entire content of the json as one single column varchar 

so basically you had to use another tool outside drill to transform the geojson file into a drill ready format (in this case csv) ? if not, could you point me into the right direction ?

thank you !
